modules/pages/client/config/pages.client.routes.js

```diff
@@ -169,7 +169,18 @@ function PagesRoutes($stateProvider) {
       footerHidden: true,
     });
   } else {
-    $stateProvider.state('home', { url: '/' });
+    $stateProvider.state('home', {
+      url: '/',
+      controller: 'HomeController',
+      controllerAs: 'stats',
+      resolve: {
```

Comment:
I don't remember now but `resolve` at route level might actually not let you render the view before this data has been fetched — therefore it becomes blocking request for the landing page which slows it down. I'd prefer if you just do `UserCountService.get()` directly in the controller in that case.

Again I'm not 100 sure if this is really blocking the view.